src/services/backend.js

```diff
@@ -8,6 +8,6 @@ if (!Backend) {
     'module specified by CONNECTOR_BACKEND was not found in /backends')
 }
 
-module.exports = new Backend({
+module.exports = new Backend(config, {
```

Comment:
We want a ctor with a common interface though. The quoter needs to know `backendUri` while fixerio does not. It seems better to keep `config` to abstract the different parameters.